[2022-04-04 06:00:02,013] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-04-04T00:00:00+00:00 [queued]>
[2022-04-04 06:00:02,018] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-04-04T00:00:00+00:00 [queued]>
[2022-04-04 06:00:02,018] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 06:00:02,018] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-04-04 06:00:02,018] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 06:00:02,025] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): enter_virtual_env> on 2022-04-04T00:00:00+00:00
[2022-04-04 06:00:02,028] {standard_task_runner.py:52} INFO - Started process 11058 to run task
[2022-04-04 06:00:02,033] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'enter_virtual_env', '2022-04-04T00:00:00+00:00', '--job-id', '37', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmp7t95fdj1', '--error-file', '/tmp/tmpq3799y23']
[2022-04-04 06:00:02,034] {standard_task_runner.py:77} INFO - Job 37: Subtask enter_virtual_env
[2022-04-04 06:00:02,061] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.enter_virtual_env 2022-04-04T00:00:00+00:00 [running]> on host 193e33218975
[2022-04-04 06:00:02,083] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=enter_virtual_env
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:00:00+00:00
[2022-04-04 06:00:02,083] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-04-04 06:00:02,083] {bash.py:158} INFO - Running command: . /app/env/bin/activate
[2022-04-04 06:00:02,090] {bash.py:169} INFO - Output:
[2022-04-04 06:00:02,096] {bash.py:177} INFO - Command exited with return code 0
[2022-04-04 06:00:02,110] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=enter_virtual_env, execution_date=20220404T000000, start_date=20220404T060002, end_date=20220404T060002
[2022-04-04 06:00:02,127] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-04-04 06:00:02,162] {local_task_job.py:146} INFO - Task exited with return code 0
